types/leaflet-offline/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for leaflet-offline 1.1
+// Project: https://github.com/robertomlsoares/leaflet-offline#readme
+// Definitions by: roblo53@hotmail.com <https://github.com/BETOXL>
```

Comment:
You're certainly free to include your email address, but most people use their name or their github username.